platform/src/main/java/platform/pageobjects/MyPage/AccountAndSecurity.java

```diff
@@ -485,6 +485,7 @@ public void Validate_Updated_Invalid_Account_Information() throws Exception {
 
 	public void Cancel_Button() throws Exception {
```

Comment:
Please use verb-Noun format for naming methods.
Cannot determine what this method does

Reply:
Resolved